packages/catalog-api/src/lib/schema.graphql

```diff
@@ -0,0 +1,246 @@
+type Query {
+  """
+  Retrieves currently indexed package metadata for all package versions.
+  """
+  package(packageName: String!): PackageInfo
+
+  """
+  Queries custom elements in the entire catalog, from the latest version of each
+  package.
+
+  Eventually this will use some sort of ranking algorithm, otherwise the order
+  will just be defined by the database implementation.
+  """
+  elements(tag: String = "latest", limit: Int): [CustomElement!]!
+
+  element(packageName: String!, elementName: String!, tag: String = "latest"): CustomElement
+
+  """
+  Validates a package, returning a list of problems
+  """
+  validate(packageName: String!, tag: String = "latest"): [String!]!
+}
+
+type Mutation {
+  importPackage(packageName: String!): PackageInfo
+
+  deletePackage(packageName: String!): Boolean
```

Comment:
Yeah. I'll do that, but also delete this from the schema for now (it's not wired up and probably shouldn't ever exist in this form).

Reply:
I removed this, but I'll add the comment to a query method that's not implemented yet. Keeping the method rather than removing specifically to get the comment started.